docs/index.zh.md

```diff
@@ -0,0 +1,50 @@
+---
+hide:
+  - navigation
+---
+
+# 滚轮俱乐部
+
+!!! 提示
```

Comment:
BTW, when you are translating these "admonitions" (docs [here](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=warning#supported-types)) you should do it like this:

```
!!! note "提示"
```